test/e2e/operator_test.go

```diff
@@ -180,6 +180,175 @@ func TestDefaultDNSSteadyConditions(t *testing.T) {
 	}
 }
 
+func TestDNSLogging(t *testing.T) {
```

Comment:
Is the move necessary in addition to the added `waitForDNSConditions` check?  Ideally, each test would ensure the cluster were in a good state before starting (thus the `waitForDNSConditions`) and leave the cluster in a good state for the next test (maybe we need another `waitForDNSConditions` in or at the end of this test?).  If re-ordering tests is still necessary for some reason, please add a comment here.
```suggestion
// TestDNSLogging verifies ...
// This test must run before other tests because ...
func TestDNSLogging(t *testing.T) {
```

Reply:
Added a comment and included reasoning in the commit message as well.

Reply:
Being devils advocate here, the fact we moved a test around and resolved a problem feels a bit fragile. If this is a requirement for this test to succeed, then I have hesitation that this test might break again.

If this is just 'helpful" to make things go smoother/quicker, then I can probably buy that.

Is it that the the other test causes so much churn, that even 5 minutes times out? These things aren't parallel, so how does it cause that much churn?

Reply:
>Is it that the the other test causes so much churn, that even 5 minutes times out?
That's correct. We have a handful of tests now that cause the daemonset to get re-rolled so the variability is much greater.

>If this is just 'helpful" to make things go smoother/quicker, then I can probably buy that.
Nail on the head. This PR is meant to unblock the existing PRs and only that. I have https://issues.redhat.com/browse/OCPBUGS-2002 open to officially deflake `TestDNSLogging` and I suspect that what we really need is a solution like `cluster-ingress-operator` where we separate serial and parallel tests.